src/main/java/org/fcrepo/auth/webac/LinkHeaderProvider.java

```diff
@@ -31,6 +34,7 @@
 import org.fcrepo.kernel.modeshape.rdf.impl.DefaultIdentifierTranslator;
 import org.fcrepo.kernel.modeshape.rdf.impl.PropertiesRdfContext;
 
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
Where is `Pair` coming from? `Pair` is gross. Can we not have a proper type here?